wordproof-timestamp/includes/NotificationHelper.php

```diff
@@ -48,43 +48,43 @@ public function checkEditScreen()
 
   public function changePostUpdateMessages($labels)
   {
-      $labels->item_updated = __( 'Your post has been updated but is at risk. Timestamp your content now to claim ownership.' );
-      $labels->item_published = __( 'Your post is published. Timestamp your content to claim ownership.' );
+      $labels->item_updated = __( 'Your post has been updated but is at risk. Timestamp your content now to claim ownership.', 'wordproof-timestamp' );
+      $labels->item_published = __( 'Your post is published. Timestamp your content to claim ownership.', 'wordproof-timestamp' );
```

Comment:
I see `'wordproof-timestamp'` being used quite often. Maybe make a constant out of this or wrap the function `__`?